packages/components/src/styles/components/form/layout.scss

```
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM LAYOUT COMPONENTS
+//
+
+@use "../../mixins/breakpoints" as *;
+
+// FORM
+
+.hds-form {
+  --hds-form-section-max-width: 672px; // default max-width used by FormSection, etc.
+
+  display: flex;
+  flex-direction: column;
+  gap: 32px;
+}
+
+// FORM HEADER & SECTION HEADER
+
+.hds-form__header,
+.hds-form__section-header {
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+// FORM SECTION
+
+.hds-form__section {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+// hasBorder = true
+.hds-form__section--has-border {
+  padding: 24px;
+  border: 1px solid var(--token-color-border-primary);
+  border-radius: var(--token-border-radius-medium);
+}
+
+// MULTI FIELD GROUP
+// Note: Uses a Layout::Flex component under the hood which includes the main layout styles
+// If responsive behavior is later added to the Layout::Flex component, this behavior should be replaced and
+// the styles should be removed
+
+.hds-form__section-multi-field-group {
+  &.hds-layout-flex--direction-row {
+    // When there are columns, stack them vertically
+    @include hds-breakpoint-below("md") {
+      flex-direction: column;
+      // per design specs: when the fields are stacked they should match the rest of the spacing and have 24px so that there isn't a weird visual rhythm
+      gap: 24px;
+      align-items: start;
+    }
+  }
+}
+
+// MULTI FIELD GROUP ITEM
+.hds-form__section-multi-field-group-item {
+  --hds-form-section-multi-field-group-item-width: unset;
+
+  width: var(--hds-form-section-multi-field-group-item-width);
+}
```

[question] would this work too (I think so, and it's simpler)? also, why `unset` and not `auto` which is the default for the CSS `width` property?
```suggestion
.hds-form__section-multi-field-group-item {
  width: var(--hds-form-section-multi-field-group-item-width, auto);
}
```